data/ddl.sql

```diff
@@ -47,6 +47,7 @@ create table sponsors (
     id uuid primary key,
     name text not null,
     website text,
+    logo text,
```

Comment:
Maybe could more intuitive to change this variable name to `logo_url` or `logo_path`.

Reply:
Sure, logo_path seems better. I'll change it